util/coordinape.js

```diff
@@ -0,0 +1,11 @@
+export const mapCoordinapeData = csvData =>
+  csvData
+    ? csvData
+        .map(item => ({
+          name: item.data[1],
+          address: item.data[2],
+          points: item.data[3],
+          epoch: item.data[5],
+        }))
+        .slice(1)
+    : []
```

Comment:
I think we need json schemas for each 3rd party integration, like in the original example I put in the gist